conans/model/build_info.py

```diff
@@ -57,6 +59,20 @@ def _filter_paths(self, paths):
         else:
             return abs_paths
 
+    def _filter_file_paths(self, paths):
+        abs_paths = [os.path.join(self.rootpath, p)
```

Comment:
Move just the abs_paths to the ``build_modules_path`` property and remove this method